packages/components/organisms/f-footer/src/tenants/es-ES.js

```diff
@@ -204,7 +204,7 @@ export default {
                     gtm: 'click_sobre_nosotros_terminos_y_condiciones'
                 },
                 {
-                    url: 'https://secure.ethicspoint.eu/domain/media/eseu/gui/106834/index.html',
+                    url: 'https://app.convercent.com/en-US/LandingPage/d8e86634-ec59-ec11-a985-000d3ab9f296',
```

Comment:
Looks like there are tenant-specific URLs for this page such as `https://app.convercent.com/es-es/LandingPage/d8e86634-ec59-ec11-a985-000d3ab9f296` that might be better to use?

Reply:
Our ticket said that the same link can be used for all tenants but I'll double check.

Reply:
Have updated 👍🏻